.changeset/tame-buses-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
```

Comment:
Should it be `minor`? since we are adding more props.

Reply:
Since its only addition of props which can happen often, we stick to patch. Only for new component or features we bump it with minor and for anything that's breaking existing consumers, we make it major

Reply:
oh I've been bumping minor for new props all this time 🙈